changelog/1478.trivial

```diff
@@ -0,0 +1 @@
+Added `--no-stdout` feature. Stdout will not shown in terminal if you use this option. Only Stderr will shown.
```

Comment:
This is a new feature (a new command-line option) so this file should be named `1478.feature`.